integration/v3_lease_test.go

```diff
@@ -438,6 +440,8 @@ func TestV3LeaseFailover(t *testing.T) {
 		t.Fatal(lresp.Error)
 	}
 
+	expectedExp = time.Now().Add(time.Duration(lresp.TTL) * time.Second)
```

Comment:
I think the problem with the proxy is that using `keepAlive()` for lease renewal doesn't return any error to the proxy such as the `request timeout ` i think. In that case, my current code doesn't return any proxy_etcd client error. so it is then the job of the proxy watch to catch the leader loss event. should we also return any errors that's due to proxy_etcd client? I am not sure how too sure how to achieve that by just using `keepAlive()`.